server/routes/dev.py

```diff
@@ -43,6 +45,49 @@ def screenshot(folder):
 
 @bp.route('/event')
 def event():
-  if not os.environ.get('FLASK_ENV') in ['autopush', 'local', 'dev']:
+  if not os.environ.get('FLASK_ENV') in [
```

Comment:
maybe we should also move event pages out of dev to stanford dc .. what do you think @beets?

Maybe for the disasters routes we can have:
`/disasters/dashboard/v0`
`/disasters/dashboard/<placeDcid>`
`/disasters/event/<eventDcid>`

This can be done in a future PR.

Reply:
since we are adding these now, let's move it here too.

Reply:
Moved